docs/docs/features/power-shaping.md

```diff
@@ -22,16 +22,38 @@ Note that this parameter only applies to Opt In consumer chains (i.e., with Top
 
 ### Capping the validator powers
 
-The consumer chain can specify a maximum fraction of the total voting power any of its validators should have.
-This is a security measure that makes it harder for a single large validator to take over a consumer chain: 
-It mitigates the risk of an Opt In chain with only a few validators being dominated by a validator with a large amount of stake.
-For example, setting this fraction to 33% would mean that no single validator can have more than 33% of the total voting power on the consumer, and thus no single validator would be able to stop the consumer by going offline.
+The consumer chain can specify a _power cap_ which corresponds to the maximum power (percentage-wise) a validator can have on the consumer chain.
```

Comment:
People seemed to have difficulty understanding the validators power cap, so adding more examples here to make it more accessible. (cc @ve6a)